package.json

```diff
@@ -23,25 +23,27 @@
     "dist/*.map"
   ],
   "scripts": {
-    "prepare": "yarn make-release && yarn build && mocha ./dist/quickjs.test.js && yarn tarball && yarn smoketest-node",
+    "prepare": "yarn prepack && mocha ./dist/quickjs.test.js && yarn tarball && yarn smoketest-node",
     "update-quickjs": "git subtree pull --prefix=quickjs --squash git@github.com:ldarren/QuickJS.git mod",
     "make-debug": "make DEBUG=1 -j 8",
-    "make-release": "which emcc && make clean && make -j 8",
+    "make-release": "embin which emcc && make clean && make -j 8",
```

Comment:
The `which emcc` is intended to fail fast if emcc is not available to avoid running `make clean` and deleting files that cannot be replaced because emcc is not available. Doesn’t need the `which` part if embin is ensuring that emcc is installed by `yarn install`.